### PR TITLE
Marimba model

### PR DESCRIPTION
**Pull request by iibrahimli**
- Add simulation package
- Add marimba xacro/URDF
- Add launch file to view model in RViz

### CODE REVIEW

File: marimbabot_sim/urdf/marimba.urdf.xacro [hunk 1]
```diff
@@ -398,4 +398,11 @@ TODO:
 <!-- TODO instantiate in standalone files -->
 <xacro:marimba name="marimba" />
 
+<link name="world"/>
```
Comment (Flova) [inline marimbabot_sim/urdf/marimba.urdf.xacro:401]: I don't know if we want to define the position of the marimba in the world statically? It likely needs to be calibrated quite often / estimated via an April tag.

File: marimbabot_sim/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+## Generate services in the 'srv' folder
+# add_service_files(
+#   FILES
+#   Service1.srv
+#   Service2.srv
+# )
+
+## Generate actions in the 'action' folder
+# add_action_files(
```
Comment (Flova) [inline marimbabot_sim/CMakeLists.txt:64]: Consider removing the examples

File: marimbabot_sim/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  rospy
```
Comment (Flova) [inline marimbabot_sim/CMakeLists.txt:12]: Are these libs really needed for the present code?

File: marimbabot_sim/launch/marimba_display.launch [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<launch>
+    <arg name="gui" default="true" />
+    <arg name="rvizconfig" default="$(find marimbabot_sim)/rviz/urdf.rviz" />
+
+    <param name="robot_description" command="xacro '$(find marimbabot_sim)/urdf/marimba.urdf.xacro'" />
+
+    <!-- <node if="$(arg gui)" name="joint_state_publisher" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" /> -->
+    <!-- <node unless="$(arg gui)" name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" /> -->
```
Comment (Flova) [inline marimbabot_sim/launch/marimba_display.launch:8]: Remove unused code

File: marimbabot_sim/package.xml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  <version>0.0.0</version>
+  <description>The marimbabot_sim package</description>
+
+  <maintainer email="ubuntu@todo.todo">ubuntu</maintainer>
+
+  <license>Apache License Version 2.0</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>rospy</build_depend>
```
Comment (Flova) [inline marimbabot_sim/package.xml:13]: Remove unused deps

File: marimbabot_sim/urdf/materials.xacro [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+  <material name="silver">
+    <color rgba="0.95 0.95 0.95 1.0"/>
+  </material>
+  
+  <material name="white">
+    <color rgba="1.0 1.0 1.0 1.0"/>
+  </material>
+  
+<!--
```
Comment (Flova) [inline marimbabot_sim/urdf/materials.xacro:52]: Remove unused code

**Review by Flova — APPROVED** [review]

**Review by Flova — APPROVED** [review]

File: marimbabot_sim/launch/marimba_display.launch [hunk 7]
```diff
@@ -4,9 +4,7 @@
 
     <param name="robot_description" command="xacro '$(find marimbabot_sim)/urdf/marimba.urdf.xacro'" />
 
-    <!-- <node if="$(arg gui)" name="joint_state_publisher" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" /> -->
-    <!-- <node unless="$(arg gui)" name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" /> -->
     <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
-    <node pkg="tf" type="static_transform_publisher" name="base_link" args="0 0 0 0 0 0 base_link marimba 100"/>
+    <node pkg="tf" type="static_transform_publisher" name="base_link" args="0 0 0 0 0 0 base_link marimba 30"/>
```
Comment (Juphex) [inline marimbabot_sim/launch/marimba_display.launch:8]: changing it to 30 is intended, right?

Comment (iibrahimli) [reply marimbabot_sim/launch/marimba_display.launch:8]: yep. not that it matters much

**Review by Juphex — APPROVED** [review]

**Pull request merged**